react-time-track/src/services/user.js

```diff
@@ -0,0 +1,14 @@
+const API_USERS = "http://localhost:3000/api/users";
+
+async function userList() {
+  const userList = await fetch(API_USERS, {
+    method: "GET",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json"
+    }
+  }).then(response => response.json());
+  return userList;
```

Comment:
What do you think about considering a block of code to control errors?. It can be of this way after line 10:
```
if (!userList.ok) {
    const { errors } = await response.json();
    console.log(errors);
    throw new Error(errors);
  }

  return userList.json();
```

Reply:
Nice. This is so useful. Thanks.